Community/Tdarr_Plugin_MC93_Migz1FFMPEG.js

```diff
@@ -259,8 +278,18 @@ const plugin = (file, librarySettings, inputs, otherArguments) => {
       // Check if codec of stream is mjpeg/png, if so then remove this "video" stream.
       // mjpeg/png are usually embedded pictures that can cause havoc with plugins.
       if (file.ffProbeData.streams[i].codec_name === 'mjpeg' || file.ffProbeData.streams[i].codec_name === 'png') {
-        extraArguments += `-map -v:${videoIdx} `;
+        extraArguments += `-map -0:v:${videoIdx} `;
+        response.infoLog += `File has an image in it as stream: ${i}, removing it! \n`;
+      } else { // if not an image stream then read it's height and width
+        try {
+          vf_scale_args = `-vf 'scale_cuda=w=${file.ffProbeData.streams[i].width}:h=${file.ffProbeData.streams[i].height}'`
```

Comment:
Why is this needed? It's scaling it to it's current resolution which seems kind of odd.

Reply:
without the `-vf scale` option, we get these errors in some files.

> [h264 @ 0x55e6568adcc0] No decoder surfaces left
Error while decoding stream #0:0: Invalid data found when processing input